### PR TITLE
WIP: check-packaging added to Makefile to verify all packaging methods

### LINKED ISSUE #110: test packaging inside CI
* [ ] python packaging
* [ ] Fedora RPM packaging (latest, rawhide)
* [ ] CentOS 7 RPM packaging

### PR DESCRIPTION
**Pull request by enriquetaso**
Fixes #110

Single make target `check-packaging` added to verify all packaging methods.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,6 +52,21 @@ check-pypi-release:
...
+install-with-pip2:
+	pip2 install --user .
+
+check-install-pip3:
+	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v $(CURDIR):/src:Z registry.fedoraproject.org/fedora:28 make install-with-pip3
+
+install-with-pip3:
+	pip3 install --user .
+
+check-packaging: rpm-in-mock-f27 rpm-in-mock-el7 rpm-in-mock-rawhide check-install-pip2 check-install-pip3
```
Comment (TomasTomecek) [inline Makefile]: `rpm-in-mock-rawhide` target is missing

File: Makefile [hunk 2]
```diff
@@ -52,6 +52,21 @@ check-pypi-release:
...
+	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v $(CURDIR):/src:Z registry.fedoraproject.org/fedora:27 make install-with-pip2
+
+install-with-pip2:
+	pip2 install --user .
+
+check-install-pip3:
+	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v $(CURDIR):/src:Z registry.fedoraproject.org/fedora:28 make install-with-pip3
+
+install-with-pip3:
+	pip3 install --user .
```
Comment (TomasTomecek) [inline Makefile]: +1

File: Makefile [hunk 3]
```diff
@@ -52,6 +52,21 @@ check-pypi-release:
...
 	PYTHONPATH=$(CURDIR) pytest -m "release_copr" ./tests/release/test_release.py
 
+check-install-pip2:
+	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v $(CURDIR):/src:Z registry.fedoraproject.org/fedora:27 make install-with-pip2
+
+install-with-pip2:
+	pip2 install --user .
+
+check-install-pip3:
+	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v $(CURDIR):/src:Z registry.fedoraproject.org/fedora:28 make install-with-pip3
```
Comment (TomasTomecek) [inline Makefile]: we would also need to check whether conu can be imported after installing it

**Comment by enriquetaso** [comment]
I don't know how to continue this PR, I'm not sure how to make conu works :thinking:

**Pull request closed without merging**